### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Mugiira-Kim**
**In this exercise I was able to;**

Use media queries to make sure my website displays correctly on desktop and mobile screens.
Parse a Figma design to create a UI.
Understand how to create UIs adaptable to different screen sizes using media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,89 +302,89 @@ <h3>
...
+
+      <div class="footershapes-desk">
           <div class="overlay">
-            <img src="rectangle.png" alt="footershapes">
+            <img src="images/contactformshapes.svg" alt="footershapes">
           </div>
-            <div class="overlay2">
+      </div>
+
+        <!--     <div class="overlay2">
```
Comment (zieeco) [inline index.html]: - [ ] Please, remove commented code, it makes your codebase not clean

File: index.html [hunk 2]
```diff
@@ -353,29 +411,29 @@ <h2>Contact me</h2>
...
+              maxlength="500">
+            </textarea>
+            <br/>
           </form>
-        </section>
-      </section>
-    </section>
+            <div class="getintouch">
+            <input class="big-button" type="submit" value="Get In Touch">
+          </div>
```
Comment (zieeco) [inline index.html]: - [ ] Please, make this a button and add the active, hover, disabled CSS pseudo classes.

clear

File: index.html [hunk 3]
```diff
@@ -245,89 +302,89 @@ <h3>
...
+                      <img src="images/github.svg" alt="github"/></a></li>
                   <li>
-                    <a href="peace"><img src="peace.svg" alt="peace" /></a>
-                  </li>
+                    <a href="peace"><img src="images/peace.svg" alt="peace" /></a></li>
                 </ul>
               </div>
-
+      
               <button class="big-button">Get my Resume</button>
```
Comment (zieeco) [inline index.html:322]: - [ ] Kindly, reduce the width of your button as it's too big on your design compared to the figma design
**YOUR DESIGN**
btn2   btn

**FIGMA DESIGN**
FIGMAA

File: index.html [hunk 4]
```diff
@@ -9,225 +9,282 @@
...
+          </div>
+
+         <div class="small-buttons">
+          <ul>
+            <li>html</li>
+          
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          </div>
```
Comment (zieeco) [inline index.html]: - [ ] Please, align your list of technologies and button accordingly following the Figma design:
**YOUR DESIGN**

LB

**FIGMA DESIGN**
noee

File: index.html [hunk 5]
```diff
@@ -9,225 +9,282 @@
...
+  <section class="aboutme">
+
+      <div class="flex1">
               <div class="mytitle">
                 <h3>
-                  About <br />
+                  About <br/>
                   Myself
                 </h3>
               </div>
```
Comment (zieeco) [inline index.html:291]: - [ ] Please, kindly remove the excess white space in your about me section, adjust your styling to be a little close to that of the Figma.

**YOUR DESIGN**

space

**FIGMA DESIGN**

NAHH

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Mugiira-Kim 


Great work on design👏 
You've made an impressive effort so far on the design, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
- N/A




## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -9,225 +9,283 @@
...
+
+  <section class="aboutme">
+
+      <div class="flex1">
               <div class="mytitle">
                 <h3>
-                  About <br />
+                  About <br/>
                   Myself
                 </h3>
```
Comment (mahbubul14) [inline index.html:290]: - [ ] As the previous reviewer suggested, Please reduce the gap between the `"About myself"` and the `"contact"` section. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
a1 | a2


**Optional:** As the previous reviewer didn't ask the following change, I kept it optional.

- [ ] Please fix the alignment of the about myself section of reduce the gap between social media icons to match with the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
ab1 | ab2

File: index.html [hunk 7]
```diff
@@ -9,225 +9,283 @@
...
-                <div class="small-buttons">
-                  <ul>
-                    <li>html</li>
-                    <li>css</li>
-                    <li>javascript</li>
-                  </ul>
-                </div>
+      <section class="project-title">
+          <h2 class="desk-title">Uber Navigation</h2>
+          <h2 class="mobile-title">Multi Post Stories</h2>
```
Comment (mahbubul14) [inline index.html:244]: - [ ] As the previous reviewer suggested, Please fix the alignment of the lists of technology and button with the content of the all cards in this work section. 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
w3 | w4

File: index.html [hunk 8]
```diff
@@ -9,225 +9,283 @@
...
-        </aside>
-        <section class="hamburger">
-          <img src="icon.jpg" alt="hamburger-icon" />
-        </section>
+
+          <aside>
+            <a href="">Portfolio</a>
+            <a href="">About</a>
+            <a href="">Contact</a>
+          </aside>
```
Comment (mahbubul14) [inline index.html:30]: **Optional:** As the previous reviewer didn't ask for that change, I kept it optional.

- [ ] Your Navbar is not matched with the Figma design. Please work on it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
n1 | n2

File: index.html [hunk 9]
```diff
@@ -9,225 +9,283 @@
...
     <!--Wrapper to wrap around cards-->
-    <section class="wrapper">
-      <section class="works-tonic">
+    
+<section class="wrapper"> 
+        
+      <!-- CARD ONE  -->
+
+ <section class="works-tonic">
+
```
Comment (mahbubul14) [inline index.html:87]: **Optional:** As the previous reviewer didn't ask for that change, I kept it optional.

- [ ] Your card's design is not matched with the Figma design. Please work on the design and also the alignment of the right & left sides.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
t | t2

File: index.html [hunk 10]
```diff
@@ -9,225 +9,283 @@
...
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="style2.css"/>
   </head>
   <body>
-    <section class="section-one">
-      <nav>
+
+<section class="header"> <!--wraps around section-one and section-two-->
+
+<section class="section-one">
```
Comment (mahbubul14) [inline index.html:20]: **Optional:** As the previous reviewer didn't ask for that change, I kept it optional.
- [ ] Your page has a horizontal scrollbar. It should be static. Please fix that issue. Your desktop version of the app is not currently optimized well. You can use [CSS Media Queries](https://gist.github.com/gokulkrishh/242e68d1ee94ad05f488) to make your page responsive well. Kindly work on it. :writing_hand: 

https://user-images.githubusercontent.com/70483835/165930455-8b8c6822-0572-465c-87e7-dd94150d669a.mp4

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Mugiira-Kim ,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

# Required Changes ♻️

- [ ] Please check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] In your `README` file, there is no "Getting Started" section. Kindly add the getting started section. This should be an example of how you may give instructions on setting up your project locally.

- [ ] Please check the comments under the review. :arrow_down: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**